frontend/app/(profile)/profile/projects/[project_id]/page.tsx

```
@@ -0,0 +1,305 @@
+                    >
+                      <div className="flex justify-between mb-2">
+                        <div className="flex items-center gap-2">
+                          <div className="bg-gradient-to-r from-purple-500 to-indigo-600 text-white h-8 w-8 rounded-full flex items-center justify-center font-bold">
+                            {bid.freelancer?.name?.charAt(0) || "F"}
+                          </div>
+                          <span className="font-medium">
+                            {bid.freelancer?.name || "Freelancer"}
+                          </span>
+                        </div>
+                        <span className="text-green-600 font-semibold bg-green-50 px-3 py-1 rounded-full">
+                          ৳{parseFloat(bid.bidding_amount).toFixed(2)}
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)